frontend_tests/node_tests/hotkey.js

```diff
@@ -278,6 +279,12 @@ function stubbing(func_name_to_stub, test_function) {
     assert_mapping('left_arrow', 'lightbox.prev');
     assert_mapping('right_arrow', 'lightbox.next');
 
+    hotkey.is_subs = return_true;
+    set_global('ui_state', {home_tab_obscured: return_true});
```

Comment:
The whole point is to toggle the state temporarily for this test, so I'm not sure we can.